examples/example-server-lib/floating_window_manager.cpp

```diff
@@ -374,8 +374,8 @@ bool FloatingWindowManagerPolicy::handle_keyboard_event(MirKeyboardEvent const*
                     (modifications.size().is_set() ? modifications.size().value() : active_window.size()).width;
 
                 modifications.top_left() = window_info.needs_titlebar(window_info.type()) ?
-                                           active_output.top_right() - as_displacement({new_width, 0}) + title_bar_height :
-                                           active_output.top_right() - as_displacement({new_width, 0});
+                                           active_output.top_right() - as_displacement(Size{new_width, 0}) + title_bar_height :
+                                           active_output.top_right() - as_displacement(Size{new_width, 0});
```

Comment:
Wait. Why are we first creating a `Size` and then running `as_displacement` on it, rather than just creating a `Displacement`?

Reply:
Because `new_width` is of type `Width`, although now that I think about it you're right. One sec...